UPGRADING.md

```diff
@@ -16,6 +16,25 @@ See [#2227](https://github.com/ruby-grape/grape/pull/2227) for more information.
 
 Rack supports a configurable limit on the number of files created from multipart parameters (`Rack::Utils.multipart_part_limit`) and raises an error if params are received that create too many files.  If you were handling the Rack error directly, Grape now wraps that error in `Grape::Execeptions::TooManyMultipartFiles`.  Additionally, Grape will return a 413 status code if the exception goes unhandled.
 
+#### Declared params can choose not to evaluate given
+
+The [`#declared(params)`] will return parameters even if depend on another parameters not given,but it will not return these if `evaluate_given=false` was set.
```

Comment:
Since the default behavior hasn't changed you don't need anything in UPGRADING. Move this into README.md and only document the new `evaluate_given` option. I can help with the English text.